coordinator/src/app_desc/genesis.rs

```diff
@@ -31,26 +31,44 @@ pub struct Genesis {
     /// Block timestamp, defaults to 0.
     pub timestamp: Option<u64>,
     /// Parent hash, defaults to 0.
+    #[serde(default)]
+    #[serde(deserialize_with = "deserialize_option_h256")]
```

Comment:
Omitting `     #[serde(deserialize_with = "deserialize_option_h256")] ` is too easy since `H256` already implements `Deserialize`. We can't use the `Deserialize` implementation since it assert using "0x" prefix.
Writing `struct H256Wrapper(H256)` for the deserialization code may be better.
@dynaxis Do you have any idea about it?

Reply:
Overall, I stil feel the current code a bit verbose than necessary. Can we create a trait named like `TryFromU8Slice`, which takes a `[u8]` parsed from a hexadecimal digits and outputs a value of the target type (such as `H256`)? Once we implement them for all types to be parsed from hexadecimal digits, we can write a wrapper type like `Hex<T: TryFromU8Slice>`, and use it here and there.

Reply:
I'll try it.